examples/wallet-export/src/components/Export.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import { IframeStamper } from "@turnkey/iframe-stamper";
+import { Dispatch, SetStateAction, useEffect, useState } from "react";
+
+interface ExportProps {
+  iframeUrl: string;
+  turnkeyBaseUrl: string;
+  iframeDisplay: string;
+  setIframeStamper: Dispatch<SetStateAction<IframeStamper | null>>;
+}
+
+const TurnkeyIframeContainerId = "turnkey-iframe-container-id";
+const TurnkeyIframeElementId = "turnkey-iframe-element-id";
+
+export function Export(props: ExportProps) {
+  const [iframeStamper, setIframeStamper] = useState<IframeStamper | null>(
+    null
+  );
+  const [iframeDisplay, setIframeDisplay] = useState<string>("none");
+
+  useEffect(() => {
+    setIframeDisplay(props.iframeDisplay);
+    return () => {
+      if (iframeDisplay === "block") {
+        setIframeDisplay("none");
+      }
+    };
+
+  }, [props.iframeDisplay]);
+
+  useEffect(() => {
+    try {
+      if (!iframeStamper) {
+        const iframeContainer = document.getElementById(TurnkeyIframeContainerId);
+        if (!iframeContainer) {
+          console.error(`Cannot create iframe stamper: no container with ID ${TurnkeyIframeContainerId} exists`);
+          return;
+        }
```

Comment:
Is this still necessary given the component is in control of creating / inserting the container into the DOM?

Reply:
Removed this check here since I made the change above allowing for `iframeContainer` to be passed as null/undefined and the check is done in the iframe stamper's constructor now.